cypress/consts.js

```diff
@@ -0,0 +1,13 @@
+export const root = "e2e-projects/cypress-next-app";
+
+export const packageJsonFile = `${root}/package.json`;
```

Comment:
tbh: I prefer sting constants to be name in UPPER_SNAKE_CASE..... I'm not sure if it'll catch on